src/repl/scala/tools/nsc/interpreter/JavapClass.scala

```diff
@@ -600,11 +600,11 @@ object JavapClass {
     def parents: List[ClassLoader] = parentsOf(loader)
     /* all file locations */
     def locations = {
-      def alldirs = parents flatMap (_ match {
+      def alldirs = parents flatMap {
```

Comment:
Not entirely sure about this change. Why would the previous version have been written in such a baroque fashion when there is an obviously simpler approach?